images/ceph/Makefile

```diff
@@ -121,3 +123,15 @@ csv: $(OPERATOR_SDK) $(YQ) ## Generate a CSV file for OLM.
 
 csv-clean: $(OPERATOR_SDK) $(YQ) ## Remove existing OLM files.
 	@rm -fr ../../cluster/olm/ceph/deploy/* ../../cluster/olm/ceph/templates/*
+
+# list-image creates list of images for offline installation
+list-image:
+	@echo "producing list of images for offline installation";\
+	# remove the file if already exists
+	[ -f ../../cluster/examples/kubernetes/ceph/images.txt ] && rm ../../cluster/examples/kubernetes/ceph/images.txt;\
```

Comment:
use a variable for `../../cluster/examples/kubernetes/ceph/images.txt` instead of duplicating it multiple times.

Reply:
done